index.html

```diff
@@ -42,6 +42,202 @@ <h4> I’am a Software Developer</h4>
+
+        <section class="card1">
+            <h2>Profesional Art Printing Data</h2>
+            <p class="jobs_p">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's standard
+            </p>
+            <div class="skills1">
+                <ul>
+                    <li>html</li>
+                    <li>javascript</li>
+                    <li>ruby</li>
+                </ul>
+            </div>
+            <a class="see_project2" href="#">See Project</a>
+        </section>
+
+        <section class="card1">
+            <h2>Profesional Art Printing Data</h2>
+            <p class="jobs_p">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's standard
+            </p>
+            <div class="skills1">
+                <ul>
+                    <li>html</li>
+                    <li>javascript</li>
+                    <li>ruby</li>
+                </ul>
+            </div>
+            <a class="see_project2" href="#">See Project</a>
+        </section>
+
+        <section class="card1">
+            <h2>Profesional Art Printing Data</h2>
+            <p class="jobs_p">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's standard
+            </p>
+            <div class="skills1">
+                <ul>
+                    <li>html</li>
+                    <li>javascript</li>
+                    <li>ruby</li>
+                </ul>
+            </div>
+            <a class="see_project2" href="#">See Project</a>
+        </section>
+
+        <section class="card1">
+            <h2>Profesional Art Printing Data</h2>
+            <p class="jobs_p">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's standard
+            </p>
+            <div class="skills1">
+                <ul>
+                    <li>html</li>
+                    <li>javascript</li>
+                    <li>ruby</li>
+                </ul>
+            </div>
+            <a class="see_project2" href="#">See Project</a>
+        </section>
+
+        <section class="card1">
+            <h2>Profesional Art Printing Data</h2>
+            <p class="jobs_p">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's standard
+            </p>
+            <div class="skills1">
+                <ul>
+                    <li>html</li>
+                    <li>javascript</li>
+                    <li>ruby</li>
+                </ul>
+            </div>
+            <a class="see_project2" href="#">See Project</a>
+        </section>
+
+    </section>
+
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
```

Comment:
- [ ] Kindly use css to add margin or padding this will make your project more semantic, It would better to remove all `<br>`  from your project 👍🏿